core/src/main/java/com/exadel/etoolbox/linkinspector/core/servlets/GenerateDataFeedServlet.java

```diff
@@ -43,46 +36,16 @@
 )
 @ServiceDescription("The servlet for manual triggering data feed generation")
 public class GenerateDataFeedServlet extends SlingSafeMethodsServlet {
-    private static final Logger LOG = LoggerFactory.getLogger(GenerateDataFeedServlet.class);
-
-    private static final String ADMIN_GROUP_ID = "administrators";
 
     @Reference
     private transient JobManager jobManager;
 
     @Override
     protected void doGet(final SlingHttpServletRequest request, final SlingHttpServletResponse response) {
-        boolean isAdmin = Optional.of(request.getResourceResolver())
-                .map(resourceResolver -> resourceResolver.adaptTo(User.class))
-                .filter(this::isAdminUser)
-                .isPresent();
-        if (isAdmin) {
-            SlingJobUtil.addJob(
-                    jobManager,
-                    DataFeedJobExecutor.GENERATE_DATA_FEED_TOPIC,
-                    Collections.emptyMap()
-            );
-        } else {
-            LOG.debug("Data feed generation was not triggered, user is not admin nor a member of '{}' group",
-                    ADMIN_GROUP_ID);
-        }
-    }
-
-    private boolean isAdminUser(User user) {
-        if (user.isAdmin()) {
-            return true;
-        }
-        try {
-            Iterator<Group> groupIterator = user.memberOf();
-            while (groupIterator.hasNext()) {
-                String groupId = groupIterator.next().getID();
-                if (groupId.equals(ADMIN_GROUP_ID)) {
-                    return true;
-                }
-            }
-        } catch (RepositoryException e) {
-            LOG.error("Failed to check if user is admin", e);
-        }
-        return false;
+        SlingJobUtil.addJob(
```

Comment:
Can we switch it to POST? Running a job *changes* the state ofthe server. It does not seem quite reasonable that we do it via GET

Reply:
Changed